File: 151001/Kononchuk/REST/src/REST.Publisher/Models/Entities/IssueTag.cs
namespace REST.Publisher.Models.Entities;

public class IssueTag
{
    public long Id { get; set; }
    
    public long IssueId { get; set; }
    public Issue? Issue { get; set; }
    
    public long TagId { get; set; }
    public Tag? Tag { get; set; }
}

File: 151001/Matsiushenko/lab4/publisher/src/main/java/by/bsuir/publisher/domain/IssueTag.java
package by.bsuir.publisher.domain;

import jakarta.persistence.Entity;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@Entity
@Table(name = "tbl_issue_tag")
@SuperBuilder
@NoArgsConstructor
@Getter
@Setter
public class IssueTag extends BaseEntity {
    @ManyToOne
    @JoinColumn(name = "issue_id")
    private Issue issue;

    @ManyToOne
    @JoinColumn(name = "tag_id")
    private Tag tag;
}
